src/main/scala/com/tresata/spark/scalding/FieldsApi.scala

```diff
@@ -31,7 +32,7 @@ class FieldsApi(val fields: Fields, val rdd: RDD[CTuple]) extends Serializable {
 
   def fieldsRDD: FieldsRDD = FieldsRDD(fields)(rdd)
 
-  def write(source: Source): Unit = CascadingRDD.saveToSource(rdd, source, fields)
+  def write(source: Source)(implicit conf: Configuration): Unit = CascadingRDD.saveToSource(rdd, source, fields, conf)
```

Comment:
if we do:
```
def write(source: Source)(implicit conf: Configuration = new Configuration): Unit = ...
```
then it will also work outside of SparkJob